app/models/miq_region.rb

```diff
@@ -2,6 +2,7 @@ class MiqRegion < ApplicationRecord
   has_many :metrics,        :as => :resource # Destroy will be handled by purger
   has_many :metric_rollups, :as => :resource # Destroy will be handled by purger
   has_many :vim_performance_states, :as => :resource # Destroy will be handled by purger
+  include MiqServer::ConfigurationManagement
```

Comment:
I don't think we should do this.
We should be using the API in the [ConfigurationManagementMixin](https://github.com/ManageIQ/manageiq/blob/1009681373bed6951f4e5bcc6a9daa9e50dcfcfd/app/models/mixins/configuration_management_mixin.rb) which is already included in `Zone` and `MiqRegion` rather than including a server concern into these models.